engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/repository-details-info.controller.js

```diff
@@ -195,6 +195,13 @@ angular.module('Bastion.repositories').controller('RepositoryDetailsInfoControll
                 $scope.save($scope.repository);
             };
 
+            $scope.clearAnsibleCollectionAuth = function () {
+                $scope.repository['ansible_collection_auth_url'] = null;
+                $scope.repository['ansible_collection_auth_exists'] = false;
```

Comment:
Since the accuracy of `ansible_collection_auth_exists` can be affected by the actions of the UI I think it may be worth removing that attribute from the API response and just calculating it on demand via a new function in this controller. Is there any other advantage in carrying that calculated value from the server side?

Reply:
I mostly copied the pattern of Username/Password field combo to not have to do too much angular.

Reply:
Let me try and update that to be UI only as there doesn't seem to be much value in the API returning it..